desktop/containerd/index.md

```diff
@@ -32,186 +65,89 @@ images** checkbox.
- Swarm: inactive
- Runtimes: runc io.containerd.runc.v2
- Default Runtime: runc
- Init Binary: docker-init
- containerd version: 9cd3357b7fd7218e4aec3eae239db1f68a5a6ec6
- runc version: v1.1.4-0-g5fd4c4d
- init version: de40ad0
- Security Options:
-  seccomp
-   Profile: builtin
-  cgroupns
- Kernel Version: 5.10.124-linuxkit
- Operating System: Docker Desktop
- OSType: linux
- Architecture: aarch64
- CPUs: 5
- Total Memory: 7.653GiB
- Name: docker-desktop
- ID: f4d28427-96df-404c-b47b-647fe5138e2a
- Docker Root Dir: /var/lib/docker
- Debug Mode: false
- HTTP Proxy: http.docker.internal:3128
- HTTPS Proxy: http.docker.internal:3128
- No Proxy: hubproxy.docker.internal
- Registry: https://index.docker.io/v1/
- Labels:
- Experimental: false
- Insecure Registries:
-  hubproxy.docker.internal:5000
-  127.0.0.0/8
- Live Restore Enabled: false
-
-```
-
-### Run a simple container
-
-```console
-$ docker run --rm hello-world
-Unable to find image 'hello-world:latest' locally
-7d246653d051: Download complete
-432f982638b3: Download complete
-46331d942d63: Download complete
-7050e35b49f5: Downloading [>                                                  ]       0B/3.208kB
-
-Hello from Docker!
-This message shows that your installation appears to be working correctly.
-...
-
-```
-
-## Run the container
+## Building multi-platform images
 
-Run the container specifying port settings:
+The term multi-platform image refers to a bundle of images that can run on different architectures.
+Out of the box, the default builder for Docker Desktop doesn't support building multi-platform images.
 
 ```console
-$ docker run -p 8080:80 -d nginx
-
-Unable to find image 'nginx:latest' locally
-b95a99feebf7: Download complete
-91d5b6827ff7: Download complete
-fc5ec3f147e4: Download complete
-5b1423465504: Download complete
-1cdde8b981f2: Download complete
-6c0b05f215c0: Download complete
-004f1937a10a: Download complete
-fd61d71c75fe: Download complete
-717bf61a04cf: Download complete
-8b6a7e0df8edbe91dfa77716a5e84ca28348f44f545a0d34c70a8987c56e63e0
-
+$ docker buildx ls | grep "DRIVER\|*"
+NAME/NODE       DRIVER/ENDPOINT             STATUS  BUILDKIT PLATFORMS
+default *       docker
+$ docker buildx build --platform=linux/amd64,linux/arm64 .
+[+] Building 0.0s (0/0)
+ERROR: multiple platforms feature is currently not supported for docker driver. Please switch to a different driver (eg. "docker buildx create --use")
 ```
 
-Confirm the Nginx container is running:
+Normally, building multi-platform images requires you to create a new builder,
+using a driver that supports multi-platform builds.
+But even then, you can't load the multi-platform images to your local image store.
 
 ```console
-$ docker ps
-CONTAINER ID   IMAGE     COMMAND                  CREATED         STATUS         PORTS                  NAMES
-93b4d60dfd08   nginx     "/docker-entrypoint.…"   3 seconds ago   Up 3 seconds   0.0.0.0:8080->80/tcp   stoic_mccarthy
+$ docker buildx create --bootstrap
```

Comment:
Wondering now if we need to show the `docker buildx create` step at all; would the error-message that's shown above be enough to illustrate what doesn't work without the integration enabled?

Reply:
Ah, gotcha. Yeah, was wondering if users would potentially run the example (and it's only there to illustrate "this is no longer needed")